### ISSUE TITLE
URGENT: NBlood crashing upon loading a savegame

### ISSUE THREAD

**Issue by OpenRift412** [nukeykt/NBlood#607]
Every few times I save and load said save, the game will crash with this error:

![image](https://user-images.githubusercontent.com/29780788/177193119-91aa2754-d00a-402a-b18f-70c8093c7b68.png)

**Comment by NY00123** [comment]
I wonder if this is specific to one of the more recent builds or not, and also if there's a known way to reproduce the problem; Say, by saving at a specific known spot and then loading.

**Comment by fhomolka** [comment]
Oh, I had this before!
The only way i managed to reproduce this is save near a bat, and then reload. Maybe quit the game and then reload.
Just to see what would happen, I remove the assert, and it appeared to work fine. But that was just a quick hack, there's probably a real solution.

**Comment by fhomolka** [comment]
@OpenRift412 Do you still have that savefile?

**Comment by OpenRift412** [comment]
> @OpenRift412 Do you still have that savefile?

This is a savegame for Death Wish, I believe this one was one of the problem saves.
[game0001.zip](https://github.com/nukeykt/NBlood/files/9221068/game0001.zip)

**Comment by fhomolka** [comment]
Cheers, I'll take a stab at this

**Comment by NY00123** [comment]
ok, so it was made with a mod. I don't know if it requires the matching Windows build and if the mod is needed. Either way, with a Linux build, I got an assertion here: `source/blood/src/seq.cpp:98` (the assertion that `pSprite->extra == nXSprite` fails).

Blood Discord member, MrJohn Reviews, also had a crash, albeit it was possibly in NotBlood instead. It was resolved by modifying notblood.cfg and appropriately setting `ModDir`. This was also done on Linux in that case.
